iceberg/iceberg-handler/src/test/results/positive/dynamic_semijoin_reduction.q.out

```diff
@@ -69,20 +69,18 @@ STAGE PLANS:
     Tez
 #### A masked pattern was here ####
       Edges:
-        Map 1 <- Reducer 5 (BROADCAST_EDGE)
```

Comment:
This execution should not be changed. This was my missing fix in HIVE-27409. I have filed a addendum patch to fix it. https://github.com/apache/hive/pull/4395